packages/openapi-code-generator/src/core/input.ts

```diff
@@ -129,6 +134,9 @@ export class Input {
           ...additionalAttributes,
           route,
           method,
+          servers: this.normalizeServers(
```

Comment:
Todo: add a new parameter to operations that have server overrides defined.

Unsure whether to roll into the main parameters object or make a new positional parameter.

Will likely suffer from naming collisions in the main object, though that's already possible today so maybe it's time to solve that properly